includes/class-dispatcher.php

```diff
@@ -95,7 +100,7 @@ public static function process_outbox( $id ) {
 		$activity = Outbox::get_activity( $outbox_item );
 
 		// Send to mentioned and replied-to users. Everyone other than followers.
-		self::send_to_interactees( $activity, $actor->get__id(), $outbox_item );
+		self::send_to_custom_inboxes( $activity, $actor->get__id(), $outbox_item );
```

Comment:
How would you feel about `activitypub_initial_inboxes()` or `activitypub_primary_inboxes()` instead of custom?

Reply:
is this really "initial" or "primary"?

Reply:
I guess it's initial in the way that it gets processed before followers 🤷 
`send_to_non_follower_inboxes`?

Reply:
I am not a fan of negations... what about something like `activitypub_special_inboxes`?

I thought about custom, because they are indirectly customized by the user through adding relays or replying to someone.

Reply:
Maybe we could go with `send_to_additional_inboxes()` to mirror the filter?